homework-submission/todo-app-week1/src/App.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react'
+import { BrowserRouter, Route, Link } from 'react-router-dom'
+import ItemsComponent from './components/ItemsComponent'
+import './App.css'
+
+class App extends Component {
+  constructor() {
```

Comment:
setting a constructor is the old way of dealing with react, because it has too much of binding and initialing the state with it's wired way `this.state = {...}` in the latest versions of React it has been updated to be:

```suggestion
  state = {...} // no binding no constructor and only state
```